feature/search/impl/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/search/impl/SearchScreen.kt

```diff
@@ -93,7 +93,11 @@ import com.google.samples.apps.nowinandroid.core.ui.NewsFeedUiState.Success
 import com.google.samples.apps.nowinandroid.core.ui.R.string
 import com.google.samples.apps.nowinandroid.core.ui.TrackScreenViewEvent
 import com.google.samples.apps.nowinandroid.core.ui.newsFeed
-import com.google.samples.apps.nowinandroid.feature.search.R as searchR
+import com.google.samples.apps.nowinandroid.feature.search.impl.RecentSearchQueriesUiState.Loading
+import com.google.samples.apps.nowinandroid.feature.search.impl.SearchResultUiState.EmptyQuery
```

Comment:
These imports reduce readability because they stop you being able to disambiguate between `RecentSearchQueriesUiState.Loading` and `SearchResultUiState.Loading`. Please remove and restore the original code.

Reply:
The IDE refactor auto-imported them for many files and I had to painstakingly revert them. These ones slipped through. :( There's probably a better way to refactor them to avoid imports.